### ISSUE TITLE
Package Manager Not Loading

### ISSUE THREAD

**Issue by BrettKnoss** [fonsp/Pluto.jl#875]
Since updating to v0.19, my package manager, and all packages will not load.  I'm using Julia 1.6.0 beta, and Vivaldi | 3.5.2115.87 (Stable channel) (64-bit), which uses JavaScript | V8 8.7.220.31, on a x86-64 bit machine running Ubuntu 20.04.1 LTS.

**Comment by fonsp** [comment]
Can you send more information?

**Comment by BrettKnoss** [comment]
I've had the same results with my book, but this is your plots sample:

![Screenshot from 2021-01-27 16-07-13](https://user-images.githubusercontent.com/73511645/106061004-6a632c80-60ba-11eb-9ceb-e8f80a58edc1.png)

I also have this error message:

```
[5948:5948:0127/160618.351734:ERROR:chrome_content_client.cc(343)] Failed to locate and load the component updated flash plugin.
Opening in existing browser session.

```
,but I think it was there when everything was working.

**Comment by bolgarbe** [comment]
Can confirm with Pluto 0.12.19 on Julia 1.6.0-beta1 with Ubuntu 20.04.1 + Chromium, Firefox and Vivaldi. Notebooks load fine but hang immediately after (i.e. can't add or run code cells). The only message I get is

`┌ Warning: Response function to message of type complete failed
└ @ Pluto ~/.julia/packages/Pluto/A96Bh/src/webserver/WebServer.jl:363`

However, it works with `Pluto.run(workspace_use_distributed=false)`.

**Comment by lungben** [comment]
Same issue for me with Windows 10 (and Julia 1.6 beta).

**Comment by fonsp** [comment]
There is no output in the terminal where you launched Pluto besides https://github.com/fonsp/Pluto.jl/issues/875#issuecomment-768990215 ?

**Comment by fonsp** [comment]
I made some changes that hopefully fix this problem, can you test it using:

```
pkg> activate --temp
pkg> add Pluto#60f6b2b
julia> import Pluto; Pluto.run()
```

and let me know whether that fixed the problem

**Comment by lungben** [comment]
It hangs for me in this line:

```
Pkg.instantiate(pkg_ctx) # we don't suppress IO for this one because it can take very long, and that would be a frustrating experience without IO
```

**Comment by lungben** [comment]
When I run the same commands from Loader.jl locally, it works:

```
pkg_ctx = Pkg.Types.Context(env=Pkg.Types.EnvCache("Project.toml")) # prepared as in Pluto
Pkg.resolve(pkg_ctx)
Pkg.instantiate(pkg_ctx)
```

**Comment by fonsp** [comment]
Can you run the whole file?
```julia
bash> julia-1.6
julia> import Pluto
julia> @time include(Pluto.project_relative_path("src","runner","Loader.jl"))
  0.184924 seconds (406.32 k allocations: 28.259 MiB, 3.90% gc time)
"/Users/fons/.julia/environments/__pluto_boot_1.5.3"
```

**Comment by lungben** [comment]
Version with lots of log statements attached, including in a monkey-patched version of `Pkg.instantiate(pkg_ctx)`.

[Loader.zip](https://github.com/fonsp/Pluto.jl/files/5889164/Loader.zip)

I am digging deeper and deeper into Pkg.instantiate, but I have not found the place yet where it hangs. Probably in a call to `Pkg.precompile(ctx)`.

**Comment by lungben** [comment]
OK, I got it working when I switch precompilation off:

```
Pkg.instantiate(pkg_ctx; update_registry=false, allow_autoprecomp=false) # we don't suppress IO for this one because it can take very long, and that would be a frustrating experience without IO
```

I still do not understand what exactly is happing here - it may even be a Pkg issue.
But for a short-term solution just switching off precompilation seems to work.

**Comment by lungben** [comment]
I'll make a PR.

**Comment by fonsp** [comment]
Awesome! Also check your email

**Comment by BrettKnoss** [comment]
The first patch didn't seem to fix it.  I didn't try with version 1.5, and I have other stuff going on right now.  I'll be sure to try @lungben 's fix later.

**Comment by fonsp** [comment]
I backported @lungben 's fix to 0.12, can you test it using:

```
# first, restart Julia

pkg> # inside the environment where you were using for Pluto:

pkg> add Pluto#8cf1689
julia> import Pluto; Pluto.run()
```

and let me know whether that fixed the problem. _(yes=👍, no=👎)_

@lungben Can you run this test too? It's different from your master branch

**Comment by BrettKnoss** [comment]
It took 71 seconds to load Plots, and 3 seconds to load the first plot, otherwise the same as normal.

**Comment by fonsp** [comment]
@BrettKnoss so https://github.com/fonsp/Pluto.jl/issues/875#issuecomment-769289355 worked?

**Comment by BrettKnoss** [comment]
Yes!

**Comment by fonsp** [comment]
Great! I have released these patches as `0.12.20`
